config/default.yml

```
@@ -958,6 +958,11 @@ Rails/ToSWithArgument:
   Safe: false
   VersionAdded: '<<next>>'
 
+Rails/TopLevelHashWithIndifferentAccess:
+  Description: 'Identifies top-level `HashWithIndifferentAccess`.'
```

Can you add the reference?

```suggestion
  Description: 'Identifies top-level `HashWithIndifferentAccess`.'
  Reference: 'https://guides.rubyonrails.org/upgrading_ruby_on_rails.html#top-level-hashwithindifferentaccess-is-soft-deprecated'
```